org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -152,4 +152,13 @@
       }
     }
   }
+
+  .collapsible-title {
```

Comment:
This style is under `.page-civicrm-vacancy` namespace, I don't know if I move to `common/` folder it'll broke another page, it's better stay under `.page-civicrm-vacancy` and move after if is necessary.